Gemfile

```diff
@@ -12,53 +8,52 @@ source 'https://rubygems.org'
 chef_version = ENV.key?('CHEF_VERSION') ? ENV['CHEF_VERSION'] : nil
 
 group :doc do
-  gem 'yard', '~> 0.9.5'
+  gem 'yard'
 end
 
 group :test do
+  gem 'berkshelf', '~> 7.0'
   gem 'rake', '~> 12.0'
-  gem 'berkshelf', '~> 6.0'
 end
 
 group :style do
-  gem 'foodcritic', '~> 10.2.0'
-  gem 'rubocop', '~> 0.39.0'
+  gem 'foodcritic'
+  gem 'rubocop'
 end
 
 group :unit do
   gem 'chef', chef_version unless chef_version.nil?
-  gem 'chefspec', '~> 6.0'
-  gem 'simplecov', '~> 0.13.0'
-  gem 'should_not', '~> 1.1'
+  gem 'chefspec'
+  gem 'should_not'
+  gem 'simplecov'
```

Comment:
These are not as important as the previous ones, but I don't know why did you remove the version constraints here too. Is this some new pattern of the community that I'm not aware of?